docs/details/dashboards/dashboard-manage-dashboards.md

```diff
@@ -0,0 +1,79 @@
+# Manage dashboards
+
+
+This section describes how to manage your PMM dashboards and the widgets on those dashboards, including:
+
+- Creating dashboard folders
+
+- Managing dashboard folders
+
+- Setting custom Home dashboard
+
+
+## Create dashboard folders
+
+Folders help you organize and group PMM dashboards, which is crucial when you have multiple dashboards or teams using the same PMM instance.
+
+!!! note alert alert-primary "Note"
+    To create a dashboard folder, you must have PMM's *Admin* privileges.
+
+To create a dashboard folder:
+
+1. On the PMM dashboards page, from the side menu, go to <i class="uil uil-plus"></i> *Dashboards > New folder*.
+
+2. Enter a unique name for your folder and click *Create*.
+
+
+## Managing dashboard folders
+
+This section describes how to delete multiple dashboards, move dashboards from one folder to another and navigate to a folder page where you can assign folder and dashboard permissions.
+
+### Delete multiple dashboards
+
+To delete multiple dashboards at once:
+
+From the side menu, go to <i class="uil uil-apps"></i> *Dashboards > Browse* and check the dashboards that you want to delete, and click *Delete*.
+
+
+
+
+![!image](../../_images/PMM_Delete_multiple_dashbaords.png)
+
+### Move dashboards from one folder to another
```

Comment:
There are 2 ways. One of them is already written by you. Other way is here: https://www.percona.com/blog/pmm-v2-30-new-dashboards-improved-dbaas-experience-free-k8s-cluster-for-testing-and-more/

Reply:
Done